### DISCUSSION TITLE
How to enable Single-Setting/One-Button Mode via QR code?

### DISCUSSION THREAD

**Discussion by rhopster** [gopro/labs#618] [Q&A] [answered]
I've read on the [GoPro community page](https://community.gopro.com/s/article/GoPro-Labs?language=en_US) that with labs you can set a mode where the operator can only start/ stop recording. But I can't seem to find what the command is. 

This is what I read on that page:
"**Single-Setting/One-Button Mode**: Enable a one-button mode where you can only start/stop capture, making it impossible to change camera mode accidentally. This is designed to capture critical events."

Does anyone know what the command is so that I can add it to a QR code?

**Comment by dnewman-gpsw** [comment, marked as answer]
It is called archive mode, weird name yes.  https://gopro.github.io/labs/control/archive/
